DynamoDS/Dynamo
# MAGN-7016: OpenFileDialog

### Purpose

This PR adds a "Open in Manual Mode" checkbox in the OpenFileDialog so that the run setting can be override when opening a file.
![openfiledialog](https://cloud.githubusercontent.com/assets/10401664/7929261/60d3f1c0-092e-11e5-91d0-e96795b6ccb4.png)



### Impact assessment

- [ ] Changes a public interface (OpenFile command)

#### Testing strategy

- Open dyn file with Automatic run setting and "Open in Manual Mode" unchecked, the run setting should be Automatic.
- Open dyn file with Automatic run setting and "Open in Manual Mode" checked, the run setting should be Manual.
- The state of the "Open in Manual Mode" should be the same the next time the OpenFileDialog is shown, i.e. if it's checked it should remain checked the next time it's shown.

### Reviewers

@sharadkjaiswal
